src/libasr/codegen/asr_to_llvm.cpp

```diff
@@ -251,6 +251,7 @@ class ASRToLLVMVisitor : public ASR::BaseVisitor<ASRToLLVMVisitor>
         llvm_utils->tuple_api = tuple_api.get();
         llvm_utils->list_api = list_api.get();
         llvm_utils->dict_api = nullptr;
+        LLVMList::enable_bounds_checking = enable_bounds_checking;
```

Comment:
causes:
```
[ 98%] Built target test_lpython
/usr/bin/ld: ../libasr/libasr.a(asr_to_llvm.cpp.o): in function `LFortran::ASRToLLVMVisitor::ASRToLLVMVisitor(Allocator&, llvm::LLVMContext&, LFortran::Platform, bool, LFortran::diag::Diagnostics&)':
/home/thirumalai/Open_Source/lpython/src/libasr/codegen/asr_to_llvm.cpp:254: undefined reference to `LFortran::LLVMList::enable_bounds_checking'
/usr/bin/ld: ../libasr/libasr.a(llvm_utils.cpp.o): in function `LFortran::LLVMList::write_item(llvm::Value*, llvm::Value*, llvm::Value*, LFortran::ASR::ttype_t*, llvm::Module&)':
/home/thirumalai/Open_Source/lpython/src/libasr/codegen/llvm_utils.cpp:990: undefined reference to `LFortran::LLVMList::enable_bounds_checking'
/usr/bin/ld: ../libasr/libasr.a(llvm_utils.cpp.o): in function `LFortran::LLVMList::write_item(llvm::Value*, llvm::Value*, llvm::Value*, llvm::Module&)':
/home/thirumalai/Open_Source/lpython/src/libasr/codegen/llvm_utils.cpp:1000: undefined reference to `LFortran::LLVMList::enable_bounds_checking'
/usr/bin/ld: ../libasr/libasr.a(llvm_utils.cpp.o): in function `LFortran::LLVMList::read_item(llvm::Value*, llvm::Value*, llvm::Module&, bool)':
/home/thirumalai/Open_Source/lpython/src/libasr/codegen/llvm_utils.cpp:2174: undefined reference to `LFortran::LLVMList::enable_bounds_checking'
collect2: error: ld returned 1 exit status
make[2]: *** [src/bin/CMakeFiles/lpython.dir/build.make:150: src/bin/lpython] Error 1
make[1]: *** [CMakeFiles/Makefile2:425: src/bin/CMakeFiles/lpython.dir/all] Error 2
make: *** [Makefile:166: all] Error 2
```

Reply:
Was this because of `static` charcteristic of the data menmeber?

Reply:
```suggestion
        bool LLVMList::enable_bounds_checking = enable_bounds_checking;
```

Reply:
```
```